source/atomics.tex

```diff
@@ -763,7 +808,7 @@
 operations on non-volatile objects become volatile. Thus, volatile qualified operations
 on non-volatile objects may be merged under some conditions. \end{note}
 
-\indexlibrary{\idxcode{atomic type}!constructor}%
+\indexlibrary{\idxcode{atomic<\placeholder{type}>}!constructor}%
```

Comment:
Likewise here, just `\indexlibrary{\idxcode{atomic}!constructor}%` would be in line with what we do elsewhere.